srcpkgs/asm6/template

```diff
@@ -0,0 +1,26 @@
+# Template file for 'asm6'
+pkgname=asm6
+version=1.6
+revision=1
+create_wrksrc=no
+hostmakedepends="unzip"
+short_desc="Simple but functional 6502 assembler"
+maintainer="rc-05 <rc23@email.it>"
+license="Public Domain"
+homepage="http://3dscapture/NES"
+distfiles="http://3dscapture.com/NES/asm6.zip"
+checksum=b37956f37815a75a6712c0d1f8eea06d1207411921c2e7ff46a133f35f0b3e1d
+nocross="Unable to strip and debug package due to unrecognized format of input file"
```

Comment:
Modified the nocross flag to give a more descriptive reason of packaging failure.

Reply:
it is nocross because you did hardcode `gcc` as compiler. Don't do that.